File: serverApi/SRC/API/Model/FornecedorModel.cs
using System;
using DOMAIN;
using Model;

namespace Model
{
    public class FornecedorModel: EntidadeBase
    {
        public string Nome { get; set; }
        public string CnpjCpf { get; set; }
        public string Email { get; set; }
        public string Telefone { get; set; }
    }
}

File: cliente/src/app/modules/fornecedores/model/fornecedor.model.ts
export class FornecedorModel {
  constructor(
      public nome?: string,
      public cnpjCpf?: string,
      public email?: string,
      public telefone?: string,
      public id?: string,
      public dataAtualizacao?: Date,
      public dataCriacao?: Date,
      public excluido?: boolean,
  ){}
}
